corehq/apps/reports/templates/reports/bootstrap3/tableau_visualization.html

```diff
@@ -80,7 +83,10 @@ <h3>
     <td data-bind="text: title"></td>
     <td data-bind="text: server"></td>
     <td data-bind="text: view_url"></td>
-    <td data-bind="text: location_safe ? 'Yes' : 'No' "></td>
+    <td>
+      <span data-bind="if: location_safe">{% trans "Yes" %}</span>
+      <span data-bind="ifnot: location_safe">{% trans "No" %}</span>
+    </td>
     <td>{% trans 'Deleted' %}</td>
```

Comment:
Since you commented [here](https://github.com/dimagi/commcare-hq/pull/34949#discussion_r1705624879), here are a couple other approaches that I think would also work:

Pretty sure since the django template evaluates first, you can just do this (though you might need to put quotes around the `{}` too)
```javascript
    <td data-bind="text: location_safe ? {% trans 'Yes' %} : {% trans 'No' %}"></td>
```

Or if you wanted to go the `if/ifnot` route, you can use virtual elements instead of spans (not that there's a problem with using spans either)
```javascript
    <td>
      <!--ko if: location_safe-->{% trans "Yes" %}<!--/ko-->
      <!--ko ifnot: location_safe-->{% trans "No" %}<!--/ko-->
    </td>
```

Reply:
Yeah, no strong preference on my end - only brought it up because it sounded like you weren't happy with the present implementation